e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -160,20 +160,17 @@ export class UIhelper {
   }
 
   async verifyRowInTableByUniqueText(
-    uniqueRowText: string | RegExp,
-    cellTexts: string[],
+    uniqueRowText: string,
+    cellTexts: string[] | RegExp[],
   ) {
-    const uniqueCell = this.page
-      .locator(UIhelperPO.MuiTableCell)
-      .locator(`text=${uniqueRowText}`);
-    await uniqueCell.scrollIntoViewIfNeeded();
-    const row = uniqueCell.locator('xpath=ancestor::tr');
-
+    const row = this.page.locator(
+      `xpath=//tr[child::td[text()='${uniqueRowText}']]`,
```

Comment:
We should avoid the use of xpaths, if the UI changes the test breaks. If possible lets us the object ID instead.

Reply:
@rnapoles-rh Locator updated.